package.json

```diff
@@ -5,6 +5,11 @@
   "main": "index.js",
   "scripts": {
     "dev": "pm2 start process-dev.json",
+    "local": "nodemon ./dist/src/app.js",
+    "build": "tsc",
+    "postinstall": "npm run build",
```

Comment:
Ideally combine with line 19 so its `"postinstall": "npm run test & npm run build",`

That will require updating eslint to lint the ts files. 

If you want me or you to follow up on eslint stuff then just make line 19 `"postinstall": "npm run build",` I'd prefer if we handle that in this PR though so it doesn't slip through the cracks.